Calculator/Calculator/Controller/CalculatorViewController.swift

```diff
@@ -56,20 +57,22 @@ class CalculatorViewController: UIViewController {
               let numberLabelText = inputNumberLabel.text else { return }
         
         if numberLabelText.count < 20 {
-            if numberLabelText == "0" {
-                inputNumberLabel.text = "0"
-            } else {
-                let formattedNumberText = numberLabelText.replacingOccurrences(of: ",", with: "") + inputNumberText
-                inputNumberLabel.text = calculatorNumberFormatter.string(for: Double(formattedNumberText))
-            }
+            let formattedNumberText = Double(numberLabelText.replacingOccurrences(of: ",", with: "") + inputNumberText)
```

Comment:
코드는 조금 길다고 느껴져요. 때문에 가독성이 떨어집니다.
또한, 형변환과 replacingOccurrences과 같은 연산도 있어 가독성이 떨어져요. 
때문에 상수로 따로 선언하고 사용하면 더 좋을 것 같아요.
```suggestion
           let a = Double(numberLabelText.replacingOccurrences(of: ",", with: "")
           let formattedNumberText = a + inputNumberText)      
```

Reply:
넵! 코멘트 참고하여 `numberLabelTextWithoutComma` 변수로 사용하였습니다!